packages/oid4vci-demo-frontend/src/configs/kvk.json

```diff
@@ -87,27 +87,24 @@
       }
     },
     "SSICredentialVerifyRequestPage": {
-      "enableRightPaneButton" : false,
+      "photoRight": "undraw_messenger_re_8bky_1.svg",
```

Comment:
I do not see this image in Figma

Reply:
https://www.figma.com/file/T5ZNMVDMWVaESbGAqc6Hek/KNB-usecase-of-the-Sphereon-Wallet?type=design&node-id=46%3A7802&mode=dev

has it